external/synth/synth/_dataset_fixtures/nudge_to_fine/after_nudging.zarr.json

```diff
@@ -0,0 +1,173 @@
+{
+    "version": "v3",
+    "schema": {
+        "coords": {
+            "time": {
+                "name": "time",
+                "dims": [
+                    "time"
+                ],
+                "value": [
+                    0,
```

Comment:
It's cleaner to hardcode the dates here than in the text fixtures IMO. This ensures that the times are consistent with the actual datasets these schemas represent.

Reply:
Makes sense to me.